components/SignInOrJoinFreeV2.js

```diff
@@ -170,11 +184,14 @@ class SignInOrJoinFreeV2 extends React.Component {
       await this.props.router.push({ pathname: '/signin/sent', query: { email: user.email } });
```

Comment:
Why are we doing this?

Reply:
This was the same way we used to do this in `SignInOrJoinFree.js`. This email is needed to be passed as a prop since we are showing the email at, 

https://github.com/opencollective/opencollective-frontend/blob/009e04903f6671e387b598d1b738c27bd920fa22/pages/signinLinkSent.js#L40-L47

So basically we have the success page with the user email shown. Maybe I am missing something, but I am not sure how the user could be redirected to the success page and show this confirmation without passing the email to that page. 🤔